### PR TITLE
Implements various changes

### PR DESCRIPTION
**Pull request by hsun3163**
1. Downloaded and revised the vcf_qc
2. Added a step in apex to generated fake_vcf
3. Added a step to generated 5 cols region list, with only somatic chr
4. Added a mechanism in gene partition to determined whether partition by gene_name or gene_id
5. Merge the covariate_merger and remove_covariates

### CODE REVIEW

File: code/data_preprocessing/phenotype/gene_annotation.ipynb [hunk 1]
```diff
@@ -210,6 +214,52 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b06cd895-342b-4206-8692-795cc39f4c20",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "[region_list_generateion]\n",
```
Comment (gaow) [inline code/data_preprocessing/phenotype/gene_annotation.ipynb:243]: typo.

File: code/data_preprocessing/covariate/BiCV_factor.ipynb [hunk 2]
```diff
@@ -96,6 +96,7 @@
     "parameter: numThreads = 8\n",
     "# Software container option\n",
     "parameter: container = \"\"\n",
+    "parameter: container_bioinfo = container\n",
```
Comment (gaow) [inline code/data_preprocessing/covariate/BiCV_factor.ipynb:99]: It doesn't really make sense to have to use two containers.

File: code/data_preprocessing/covariate/BiCV_factor.ipynb [hunk 3]
```diff
@@ -85,7 +85,7 @@
     "# The VCF file, required by APEX\n",
     "# https://github.com/hsun3163/neuro-apex/issues/1\n",
     "# https://github.com/cumc/xqtl-pipeline/issues/138\n",
-    "parameter: vcfFile = path\n",
+    "#parameter: vcfFile = path\n",
```
Comment (gaow) [inline code/data_preprocessing/covariate/BiCV_factor.ipynb:88]: Please remove this line.

**Comment by gaow** [comment]
@hsun3163 I've merged it for now but let's address to my comments in your next PR.

File: code/data_preprocessing/covariate/BiCV_factor.ipynb (hunk 2 above)
Comment (hsun3163) [reply code/data_preprocessing/covariate/BiCV_factor.ipynb:99]: If we dont use two containers, there are two other ways:
1. Add the bgzip and tabix to apex
2. Run the fake vcf as a explicit step.
I wonder which is better?

Comment (gaow) [reply code/data_preprocessing/covariate/BiCV_factor.ipynb:99]: I would go for the first proposal.

**Pull request merged**